roles/cdi/README.md

```diff
@@ -0,0 +1,17 @@
+# Ansible role: cdi
+
+This role deploys the CDI controller.
+
+### Variables
+| Variable        | Default Value           | Description  |
+|:------------- |:-------------|:----- |
+| cdi_image_namespace | golden-images | The namespace into which the CDI components should be installed |
+| action | provision | The action of provisioning or deprovisioning CDI |
+| cdi_repo_tag | jcoperh | CDI docker hub repo tag |
+| cdi_release_tag | latest | CDI docker hub release tag |
+
+## To deploy the CDI Controller run as follows:
+
+```
+ansible-playbook -i inventory -e action=provision -e cdi_image_namespace=golden playbooks/cdi.yml
+```
```

Comment:
Please switch Deployment and Variables sections. Usually, a command goes first, and then explanation of all variables and options. That's how the Content Services team at Red Hat writes docs.

Reply:
fixed